### DISCUSSION TITLE
Nats Request claims!

### DISCUSSION THREAD

**Discussion by Pratik-WS** [nats-io/nats.deno#724] [Q&A] [answered]
Hello there, I am trying to integrate nats auth_callout in my web app which have chat feature for that i am using nats and want to use nats auth_callout for the same. So is there any way to generate request claims automatically using NSC command or I have to generate is manually in a json format?

**Comment by aricart** [comment, marked as answer]
well - the idea is that you could use NSC or any tooling you create to generate a sentinel JWT, if your initial connect is in an "operator" (a delegated authentication) context. But the callout should be very adept at generating JWTs as that is what it has to do ultimately. 

While this is still a WIP you can take a look on how a callout works by looking at:

https://github.com/synadia-io/orbit.js/tree/authcallout/callout
